Create README.md

jay270804 opened the pull request:
This PR closes #1 

**Added README.md**

Kindly review it and drop suggestion for changes if necessary.

jay270804 committed:
Create README.md

Added README.md

HarshDobariya79 commented on README.md:
```diff
@@ -0,0 +1,55 @@
+### Prerequisites
+Before you begin, ensure you have the following installed on your machine:
+
++ **Node.js (version 14.x or later)**
```
The nodejs version will be specifically 20.x to avoid dependency issues. We can also add some info about NVM here.

jay270804 replied:
Sure, I will resolve that soon.

HarshDobariya79 commented on README.md:
```diff
@@ -0,0 +1,55 @@
+# Install dependencies that will be mentioned after this quick setup section
+npm install
+
+# Create .env file and add environment variables
```
Can we avoid creating the `.env` and rename the existing `env.example` file to make sure all the env variables are present in the `.env`?

jay270804 replied:
Will look at that too, soon!

HarshDobariya79 commented on README.md:
````diff
@@ -0,0 +1,55 @@
+```bash
+
+# Clone the repository 
+git clone https://github.com/your-username/checkmark-backend.git
````
We can add a section that guides first time contributors on how to fork the repository.

jay270804 replied:
Sure. But don't you think this particular guide should be added to contributing.md ?

HarshDobariya79 replied:
Yes, that's correct and that is already present in the `CONTRIBUTING.md`. But it'll be better to just add the one step here as well since it is easily visible to beginners.

jay270804 replied:
Alright, I will get it done !

HarshDobariya79 reviewed (commented):
Thank you for the quick PR. I've added some suggestions. Please share your thoughts.

jay270804 committed:
Update README.md

Added: 
+ the forking guide 
+ updated node version 
+ added the instruction to  rename env.example to .env

jay270804 commented:
I did all the revisions.

HarshDobariya79 commented on README.md:
```diff
@@ -0,0 +1,57 @@
+
+# Copy env.example to .env and edit with your variables
+cp env.example .env
+sed -i 's/your_mongodb_uri/<your_actual_mongodb_uri>/' .env
```
We can omit this part from the script.

jay270804 replied:
Line 30-33 ?

HarshDobariya79 reviewed (commented):
Thank you for implementing the requested changes. Just one little change I've mentioned.

jay270804 committed:
Update README.md 

+ omitted the lines.

HarshDobariya79 reviewed (approved):
Thank you for implementing the required changes @jay270804.
